index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="UTF-8" >
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
+  </head>
+    <header class="container">
+      <div class="row" id="header">   
+        <nav id="nav-bar" class="navbar navbar-expand-lg navbar-light bg-light">
+         <img 
+         id="header-img"
+         src="/img/logo.jpg"
+         height="100px"
+      >
+         <a class="navbar-brand" href="#">Company XYZ</a>
+         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+           <span class="navbar-toggler-icon"></span>
+         </button>
+         <div class="collapse navbar-collapse" id="navbarNavDropdown">
+           <ul class="navbar-nav">
+             <li class="nav-item active">
+               <a class="nav-link" href="#welcome-section">Home</a>
+             </li>
+             <li class="nav-item">
+               <a class="nav-link" href="#features">Features</a>
+             </li>
+             <li class="nav-item">
+               <a class="nav-link" href="#pricing">Pricing</a>
+             </li>
+           </ul>
+         </div>
+       </nav>
+    </div>
+    </header>   
+
+  <body>
+    
+      <div id="main" class="jumbotron">
+            <div class="container"> 
+             
+           <div class="row" id="welcome-section">
+                 <h1>Welcome to our company</h1>
+           </div>
+
+          <div>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+              <div class="form-group">
+                <label for="exampleInputEmail1">Email address</label>
+                <input id="email" type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email" required>
+                <small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
+              </div>
+              <input name="submit" id="submit"  type="submit" value="Get Started">
+            </form>
+          </div>
+            
+           
+         
+          
+          <div id="pricing"> 
+                <h1>There are different pricing</h1>
+                <ul>
```

Comment:
Please change the h1 to  h2 since in one HTML there should be one h1 element for SEO and semantic reasons.